_includes/tutorials/hopping-windows/ksql/code/src/statements.sql

```diff
@@ -8,8 +8,8 @@ CREATE STREAM TEMPERATURE_READINGS (ID VARCHAR, TIMESTAMP VARCHAR, READING BIGIN
 CREATE TABLE TRIGGERED_ALERTS AS
     SELECT
         ID,
-        TIMESTAMPTOSTRING(WindowStart(), 'HH:mm:ss') AS START_PERIOD,
-        TIMESTAMPTOSTRING(WindowEnd(), 'HH:mm:ss') AS END_PERIOD,
+        TIMESTAMPTOSTRING(WINDOWSTART, 'HH:mm:ss') AS START_PERIOD,
```

Comment:
`WindowStart` and `WindowEnd` UDAFs have been removed (https://github.com/confluentinc/ksql/pull/4459) and now the `WINDOWSTART` and `WINDOWEND` are available as columns.